spaces/domains.json

```
@@ -211,4 +211,5 @@
   "gov.zoocoin.cash": "zoocoin.eth",
   "vote.nftsnafu.org": "nftsnafu.eth",
   "vote.fairmoon.community": "fairmoon.eth"
+  "dao.cybertime.fund": "cybertime.eth
```

A comma and a quote are missing at the end of the line above your sub-domain.
```suggestion
  "vote.fairmoon.community": "fairmoon.eth",
  "dao.cybertime.fund": "cybertime.eth"
```